include/seastar/core/file.hh

```diff
@@ -100,6 +100,13 @@ struct file_open_options {
     static constexpr uint32_t min_extent_size_hint_alignment{128u << 10}; // 128KB
 };
 
+struct group_details {
+    char *group_name;
+    char *group_password;
+    uint64_t group_gid;
+    char **group_member;
+};
```

Comment:
This is just a copy of "struct group" from <grp.h>, so why not use the original?

Reply:
It's actually bad to use the original (or a copy) since they're pointers into nowhere. We need a struct with values, not pointers of questionable lifetime.

Reply:
Plus documentation.